charts/fluentd/templates/daemonset.yaml

```diff
@@ -31,6 +31,26 @@ spec:
               secretKeyRef:
                 name: {{ template "fullname" . }}
                 key: elasticsearch_port
+          - name:  FLUENT_ELASTICSEARCH_SCHEME
+            valueFrom:
+              secretKeyRef:
+                name: {{ template "fullname" . }}
+                key: elasticsearch_scheme
+          {{ if .Values.elasticsearch.username }}
```

Comment:
Stupid question: Why do we need to check if these values are present? I didn't check them in the `webapp` helm chart for example.